test/unit/content-src/components/DiscoveryStreamComponents/DSContextFooter.test.jsx

```diff
@@ -28,7 +28,12 @@ describe("<DSContextFooter>", () => {
     assert.isOk(wrapper.find(".story-footer"));
   });
   it("should render an engagement status if no badge and spoc passed", () => {
-    wrapper = mount(<DSContextFooter engagement={engagement} />);
+    wrapper = mount(
+      <DSContextFooter
+        display_engagement_labels={true}
```

Comment:
May be add a test that checks engagement label is not rendered when display_engagement_labels is false even though engagement is set